tests/dispatch/test_http2.py

```diff
@@ -204,3 +204,11 @@ def test_http2_reconnect():
     for setting_code, changed_setting in settings.changed_settings.items():
         assert isinstance(changed_setting, h2.settings.ChangedSetting)
         assert changed_setting.new_value == expected_settings[setting_code]
+
+
+async def test_http2_live_request(backend):
+    async with AsyncClient(backend=backend) as client:
+        resp = await client.get("https://nghttp2.org/httpbin/anything")
```

Comment:
Could https://github.com/encode/httpx/pull/412 help with test?

Reply:
Definitely, it's a lot further out than we want this work completed though. Hopefully soon!